packages/nextra-theme-docs/src/index.tsx

```diff
@@ -21,22 +21,7 @@ import { getComponents } from './mdx-components'
 import { ActiveAnchorProvider, ConfigProvider, useConfig } from './contexts'
 import type { PageTheme } from './constants'
 import { DEFAULT_LOCALE, PartialDocsThemeConfig } from './constants'
-import { getFSRoute, normalizePages, renderComponent } from './utils'
-
-function useDirectoryInfo(pageMap: PageMapItem[]) {
-  const { locale = DEFAULT_LOCALE, defaultLocale, asPath } = useRouter()
-
-  return useMemo(() => {
-    // asPath can return redirected url
```

Comment:
since asPath will be redirected URL, normalize pages will never return correct pageItem